website/developer-docs/setup/full-dev-environment.md

```diff
@@ -54,6 +60,10 @@ This will immediately update the UI with any changes you make so you can see the
 
 To format the frontend code, run `make web`.
 
-## Running
+## Running authentik
 
 Now that the backend and frontend have been setup and built, you can start authentik by running `ak server`. authentik should now be accessible at `http://localhost:9000`.
+
+:::info
+If you need to define a password for the default admin (called **akadmin**), you can manually enter the `/if/flow/initial-setup/` path in the browser address bar to launch the initial flow.
```

Comment:
I don't think the `If you need to define` makes sense here, this is more of a required step

Reply:
```suggestion
To define a password for the default admin (called **akadmin**), you can manually enter the `/if/flow/initial-setup/` path in the browser address bar to launch the initial flow.
```